src/app/main.cpp

```diff
@@ -104,6 +104,10 @@ void adjustFileDescriptorLimit();
 // Main
 int main(int argc, char *argv[])
 {
+#ifdef DISABLE_GUI
+    setvbuf(stdout, nullptr, _IONBF, 0);
```

Comment:
Is it a really good idea to completely disable buffering? Wouldn't it be better to flush it where it is really necessary, as previously suggested by @sledgehammer999?

Reply:
>Is it a really good idea to completely disable buffering? 

Quoting the commit message body:
>The messages printed out via stdout is usually important and short so there is no reason to buffer them.

Also IMO stdout buffering is helpful when app use it to pipe data to other programs for further processing. Not so much when printing messages to user.

Reply:
> Also IMO stdout buffering is helpful when app use it to pipe data to other programs for further processing. Not so much when printing messages to user.